src/Core/Comparer.cs

```diff
@@ -24,21 +24,21 @@ public static class Comparer
         public static EqualityComparer<T, TResult> ContraMap<T, TResult>(this IEqualityComparer<T> comparer, Func<TResult, T> contraMapper)
             => new EqualityComparer<T, TResult>(comparer, contraMapper);
```

Comment:
Move arrow to the end of the function declaration to be consistent with the rest of the project.